### PR TITLE
Fix #1298: Lab Overview Title Not Displaying

### LINKED ISSUE #1298: Lab Overview title not displaying
Tab shows unknown on click of the Lab overview menu.

### Steps to reproduce this ###

Click on the following **Procedures > Lab Overview** 

### Expected behaviour ###
Lab overview tab should open with title Lab overview

### Actual behaviour ###
Shows unknown title tab bar

screen shot 2018-11-24 at 7 44 51 am

### PR DESCRIPTION
**Pull request by jonathanzhang53**
## Description
The issue was that the title was never declared in the respective php file. Just added the tag and it is fixed.

## Screenshot
image

### CODE REVIEW

File: interface/patient_file/summary/labdata.php [hunk 1]
```diff
@@ -81,6 +81,7 @@
 // ####################################################
 echo "<html><head>";
 ?> 
+<title><?php echo xlt('Lab Overview'); ?></title>
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
```
Comment (muarachmann) [inline interface/patient_file/summary/labdata.php]: Yop this does the trick. Good catch

Comment (jonathanzhang53) [reply interface/patient_file/summary/labdata.php]: Thanks.

Comment (aethelwulffe) [reply interface/patient_file/summary/labdata.php]: If "lab overview" does not need to be echoed onto the form (it is already in the tab) then set the title to -hidden.

**Comment by aethelwulffe** [comment]
Mua approved.  AS I already tested my big merges, I guess we can do this tiny one.

**Comment by Manvi07** [comment]
Hi @jonathanzhang53 
Is this issue resolved? What I'm getting is something different, the heading is "Notice". 
screenshot from 2019-02-28 16-16-57

**Comment by muarachmann** [comment]
@Manvi07 yes this issue will be resolved once this PR would be merge. Try working on other issues thanks

File: interface/patient_file/summary/labdata.php (hunk 1 above)
Comment (muarachmann) [reply interface/patient_file/summary/labdata.php]: @jonathanzhang53 i think #1331 needs this

Comment (jonathanzhang53) [reply interface/patient_file/summary/labdata.php]: Cool I'll check it out.

**Pull request merged**